src/sonic-yang-models/yang-models/sonic-buffer-pool.yang

```diff
@@ -57,6 +61,17 @@ module sonic-buffer-pool {
                     type uint64;
                     description "Buffer Pool Xoff Threshold (in Bytes)";
                 }
+
+                leaf percentage {
+                    type uint8;
+                    description "Buffer Pool percentage";
```

Comment:
Can you provide more details? For example, buffer size * percentage / 100

Reply:
Updated.